### PR TITLE
add: test case for WebClientPluginTest and WebClientResponsePluginTest(fixes #837)

### LINKED ISSUE #837: Add test case for soul-plugin-httpclient
- [x] WebClientPlugin @Trafalgar-YuI 
- [x] WebClientReponsePlugin @Trafalgar-YuI 
- [ ] NettyHttpClientPlugin @Trafalgar-YuI 
- [ ] NettyClientReponsePlugin @Trafalgar-YuI

### PR DESCRIPTION
**Pull request by Trafalgar-YuI**
…t(fixes #837)

// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-common/src/main/java/org/dromara/soul/common/enums/PluginEnum.java [hunk 1]
```diff
@@ -82,6 +82,11 @@
      */
     DIVIDE(50, 0, "divide"),
 
+    /**
+     * WebClient plugin enum.
+     */
+    WEB_CLIENT(51, 0, "webClient"),
```
Comment (tydhot) [inline soul-common/src/main/java/org/dromara/soul/common/enums/PluginEnum.java:88]: why and this??

Comment (Trafalgar-YuI) [reply soul-common/src/main/java/org/dromara/soul/common/enums/PluginEnum.java:88]: to compare with webClientPlugin.getOrder() and webClientPlugin.named()

Comment (tydhot) [reply soul-common/src/main/java/org/dromara/soul/common/enums/PluginEnum.java:88]: don't modify here for test

Comment (Trafalgar-YuI) [reply soul-common/src/main/java/org/dromara/soul/common/enums/PluginEnum.java:88]: oh, I will change back

**Review by tydhot — APPROVED** [review]

**Pull request merged**